src/views/SchemaForm.vue

```diff
@@ -207,7 +207,7 @@ export default {
           .then(data => {
               this.formValidated = true
               this.faultyFields = []
-              const errors = data.report.tables[0].errors
+              const errors = data.report.tasks[0].errors
```

Comment:
can we do something retro-compatible? it will make the migration easier.

Reply:
Sure we can. I work on it.

Reply:
csv-gg now handles legacy and new Validata validation report with https://github.com/etalab/csv-gg/pull/67/commits/64cfde6d8a5e304a44b478acff481ff0e84e905e